website/blog/2023-11-15-how-wireguard-makes-zero-trust-acheivable/item.md

```diff
@@ -0,0 +1,158 @@
+---
+title: How Wireguard finally makes a zero trust future achievable
+description: “Thanks to a few advancements (mainly Wireguard), zero trust will soon go from buzzword to reality.”
+slug: 2023-11-15-how-wireguard-makes-zero-trust-acheivable
+authors:
+    - name: Jens Langhammer
+      title: CTO at Authentik Security Inc
+      url: https://github.com/BeryJu
+      image_url: https://github.com/BeryJu.png
+tags:
+    - authentik
+    - zero trust
+    - Wireguard
+    - NIST
+    - Okta
+    - breaches
+    - SSO
+    - security
+    - identity provider
+    - authentication
+hide_table_of_contents: false
+---
+
+> **_authentik is an open source Identity Provider that unifies your identity needs into a single platform, replacing Okta, Active Directory, and auth0. Authentik Security is a [public benefit company](https://github.com/OpenCoreVentures/ocv-public-benefit-company/blob/main/ocv-public-benefit-company-charter.md) building on top of the open source project._**
+
+---
+
+Buzzwords are the scourge of the tech industry – reviled by developers, pushed by vendors, and commanded by executives.
+
+All too often, a buzzword is the first signal of rain ([or worse](https://media.licdn.com/dms/image/C4E12AQGspNcRlqpg0A/article-inline_image-shrink_1000_1488/0/1593238107360?e=1700092800&v=beta&t=SCKZ-7W_R9swJPwEpBB35OsVc0jE093ylcjxFPm6FZc)): Marketers have created a trend; vendors are using the trend to explain why you need to buy their software right now; executives are worried about a problem they didn’t know existed before they read that Gartner report; and the downpour rains on developers.
+
+“_Implement zero trust!_”
+
+“_Why aren’t we shifting left?_”
+
+“_Are we resilient? Well, can we get more resilient?_”
+
+After a while, buzzwords start to look like trojan horses, and the invading army feels like a swarm of tasks that will result in little reward or recognition. It’s tempting to retreat to cynicism and to ignore every Term™ that comes your way.
+
+But this can be risky. For better or worse, good ideas inevitably get branded, and if you want to keep up, you need to see past the branding – even if it involves stripping away the marketing fluff to see the nugget of an idea within.
+
+There’s no better example of this than zero trust. In this post, we’ll briefly explore the term's history, explain how it became such an untrustworthy buzzword, and argue that thanks to a few advancements (mainly Wireguard), zero trust will soon go from buzzword to reality.
```

Comment:
Suggestion: s/of this/of a good idea buried under buzzword bingo/. (Or something similar)

I'm not a huge fan of pronouns with weak antecedents.  :-)